docs/docs/cmd/spo/list/list-label-get.md

```diff
@@ -34,3 +37,15 @@ Gets label set on the list with id _cc27a922-8224-4296-90a5-ebbc54da2e85_ locate
 ```sh
 m365 spo list label get  --webUrl https://contoso.sharepoint.com/sites/project-x --listId cc27a922-8224-4296-90a5-ebbc54da2e85
 ```
+
+Gets label set on the list with server relative url _sites/project-x/Documents_ located in site _https://contoso.sharepoint.com/sites/project-x_
```

Comment:
we could also do the same here not to duplicate the values from the example line 👍